growth/too/templates/plan_manual.html

```diff
@@ -67,6 +67,39 @@
                     0 for standard imaging and readout
                     </small>
                 </div>
+                <div class="form-group col-sm-4">
+                    <label for=mode_num>Number of EWR Images (ZTF Only)</label>
+                    {{ render_field(form.ewr_num_images, placeholder='Number of EWR images') }}
+                    <small id="balanceHelpBlock" class="form-text text-muted">
+                    1 for standard imaging and readout
+                    </small>
+                </div>
+		
+            </div>
+            </div>
+            <div class=form-row>
+                <div class="form-group col-sm-6">
+                    <label for=plan_name>Queue name</label>
+                    {{ render_field(form.queue_name, placeholder='Queue name') }}
+                </div>
+                <div class="form-group col-sm-6">
+                    <label for=subprogram_name>Subprogram name</label>
+                    {{ render_field(form.subprogram_name, placeholder='Subprogram name') }}
```

Comment:
Is freeform text really accepted here, or are there only specific options that are accepted?

Reply:
Yes, freeform text is accepted--this isn't actually new code though, I accidentally duplicated it.